src/components/toggle/index.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import classnames from 'classnames';
+import modifiers from '../../utils/modifiers';
+import './toggle.css';
+
+/**
+ * Shows a toggle button for code panel
+ */
+const Toggle = ({
+  id = '',
+  checked,
+  enabled = true,
+  title,
+  onChange,
+  className = '',
+}) => (
+  <div
+    className={modifiers(classnames('pipeline-toggle', className), {
```

Comment:
Not sure why this css class is named as `pipeline-toggle` given that this toggle is a general component

Reply:
I think `pipeline-` is the general prefix for everything in Viz - we've talked before about changing it to `kv` to avoid this confusion but for now it should be there.

Reply:
I see, thanks for clarifying @bru5 . I don't think this has been communicated within the team before, good to know about this, and on checking the code I do see this as a pattern.

I wonder what was the reason behind using `pipeline-` as the general prefix? This is definitely confusing and worth a future ticket and discussion about how to go forward with this.